src/utils/components/VirtualMachineDescriptionItem/VirtualMachineDescriptionItem.tsx

```diff
@@ -74,16 +74,18 @@ const VirtualMachineDescriptionItem: FC<VirtualMachineDescriptionItemProps> = ({
             default: editOnTitleJustify ? 'justifyContentSpaceBetween' : 'justifyContentFlexStart',
           }}
         >
-          <FlexItem>
-            <DescriptionItemHeader
-              bodyContent={bodyContent}
-              breadcrumb={breadcrumb}
-              descriptionHeader={descriptionHeader}
-              isPopover={isPopover}
-              label={label}
-              moreInfoURL={moreInfoURL}
-            />
-          </FlexItem>
+          {(bodyContent || breadcrumb || descriptionHeader || label || moreInfoURL) && (
```

Comment:
can u please explain the condition seems a bit strange the || between hard values

Reply:
We don't want to render the header if none of these props are defined.